docs/machine-learning/ml-proposed-process.md

```diff
@@ -79,7 +79,7 @@ For more information on experimentation, refer to the [experimentation](ml-exper
 
 ### Model Operationalization
 
-* Taking "experimental" code and preparing it, so it can be deployed. This includes data pre-processing, featurization code, training model code (if required to be trained using CI/CD) and model inference code.
+* Taking "experimental" code and preparing it, so it can be deployed. This includes data pre-processing, code to generate data features, training model code (if required to be trained using CI/CD) and model inference code.
```

Comment:
Is "featurization" a term in machine learning?

- https://learn.microsoft.com/en-us/azure/machine-learning/v1/how-to-configure-auto-features?view=azureml-api-1
- https://learn.microsoft.com/en-us/azure/databricks/machine-learning/preprocess-data/transfer-learning-tensorflow

Reply:
I'll change it back